### PR TITLE
MEN-5093: include auditlogs to default ME setup

### PR DESCRIPTION
**Pull request by 0lmi**
https://tracker.mender.io/browse/MEN-5093

### CODE REVIEW

**Comment by 0lmi** [comment]
@mender-test-bot start pipeline

File: demo [hunk 1]
```diff
@@ -10,7 +10,7 @@ DOCKER_COMPOSE_PROJECT_NAME=${DOCKER_COMPOSE_PROJECT_NAME:-$(tr -d ".-" <<<${PWD
 EXTRA_FILES=""
 
 CLIENT_FILES="-f docker-compose.client.yml -f docker-compose.client.demo.yml"
-ENTERPRISE_FILES="-f docker-compose.enterprise.yml -f docker-compose.monitor.yml"
+ENTERPRISE_FILES="-f docker-compose.enterprise.yml -f docker-compose.monitor.yml -f docker-compose.auditlogs.yml -f docker-compose.auditlogs.connect.yml"
```
Comment (mzedel) [inline demo:13]: with `docker-compose.config.yml` included by default, this should also include `docker-compose.auditlogs.config.yml`

**Review by lluiscampos — COMMENTED** [review]
A part from what Manuel pointed out, I see that `devicemonitor` is also missing from [the production template](https://github.com/mendersoftware/integration/blob/master/production/run#L23). Shouldn't we align these?

**Comment by 0lmi** [comment]
> A part from what Manuel pointed out, I see that devicemonitor is also missing from the production template. Shouldn't we align these?

@lluiscampos it's actually in place. `docker-compose.monitor.yml` contains `mender-devicemonitor`.

**Comment by 0lmi** [comment]
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/375775272

**Comment by lluiscampos** [comment]
> > A part from what Manuel pointed out, I see that devicemonitor is also missing from the production template. Shouldn't we align these?
> 
> @lluiscampos it's actually in place. `docker-compose.monitor.yml` contains `mender-devicemonitor`.

Yep, you have added the files now in your force-push. Thanks :+1:

**Comment by lluiscampos** [comment]
Remember to follow-up upon merge updating the expected number of containers in mender-docs. [Here](https://raw.githubusercontent.com/mendersoftware/mender-docs/master/07.Server-installation/03.Installation-with-docker-compose/docs.md) and [here](https://raw.githubusercontent.com/mendersoftware/mender-docs/master/07.Server-installation/03.Installation-with-docker-compose/02.Upgrading-from-OS-to-Enterprise/docs.md).

**Review by lluiscampos — APPROVED** [review]

**Comment by 0lmi** [comment]
Yes. I finally got what you meant.

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Pull request merged**